doc/source/usage.rst

```diff
@@ -59,7 +59,7 @@ A ``.travis.yml`` testing a role named foo1 with the Docker driver.
     - sudo apt-get -qq update
     - sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes -y docker-engine
   install:
-    - pip install molecule
+    - pip install molecule docker
```

Comment:
I think we want to make this a comment.  This assumes everyone will be using docker, when we support a few drivers.  Could we change this to a comment?

```
 - pip install molecule
# - pip install required driver (e.g. docker, python-vagrant, shade)
```